spec/models/defence_request_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe DefenceRequest, type: :model do
+  subject { DefenceRequest.new }
```

Comment:
But if you don't explicitly set subject, then it will be set to the default for any nested described blocks, so we surely need this?